backend/docker_utils/compose_utils.py

```
@@ -0,0 +1,279 @@
+        logger.info("All services stopped")
+        return result.returncode == 0
+    except Exception as e:
+        logger.error(f"Error stopping services: {e}")
+        return False
+
+
+# 1. Update compose_utils.py functions
+
+
+# backend/docker_utils/compose_utils.py
+def run_docker_compose_command(
```

Ruff (0.8.2)

Redefinition of unused `run_docker_compose_command` from line 9

(F811)